src/Viewer.tsx

```diff
@@ -355,6 +358,16 @@ function Viewer(): ReactElement {
   );
 
   // DATASET LOADING ///////////////////////////////////////////////////////
+
+  const handleProgressUpdate = useCallback(
+    (complete: number, total: number): void => {
+      startTransition(() => {
+        setDatasetLoadProgress(Math.round((complete / total) * 100));
+      });
+    },
+    [setDatasetLoadProgress]
+  );
```

Comment:
I believe state setters from `useState` are guaranteed to be stable? If so, should this dependency be removed?

Reply:
This comment was motivated by seeing this function (`handleProgressUpdate`) in a `useCallback` dependency array further down, thinking for a sec it was a `useEffect`, and wondering what that might mean semantically (change datasets whenever the function that changes the progress bar changes...?). Code makes a lot more sense when you read it right 🙃 and this is ultimately much less important a question than I thought.

Reply:
Still agree that this dependency can be removed!